examples/reminders/lib/base/di/app_dependencies.dart

```diff
@@ -58,6 +60,11 @@ class _AppDependenciesState extends State<AppDependencies> {
     shellNavigatorKey = GlobalKey<NavigatorState>();
   }
 
+  @override
+  void dispose() {
+    super.dispose();
+  }
```

Comment:
Overriding the method is not needed here as it's implementation is the underlying implementation from super class.